test/unittests/test_recordstore.py

```diff
@@ -457,6 +461,9 @@ def test_create_project(self):
     def test_project_info(self):
         self.assertEqual(self.store.project_info("TestProject")["name"], "TestProject")
 
+    def test_clear(self):
+        pass  # raises UserWarning
```

Comment:
Yes, will do. Any opinions on my other question, whether upgrading from an older Sumatra version can cause problems due to the table `django_store_record_output_data` having been removed?